index.md

```diff
@@ -18,10 +18,31 @@ Please submit a pull request if you have something to add!
 
 * [Stephen Holiday](https://stephenholiday.com)
 * [Tony Dong](tony-dong.com)
+* [Michael Shao](http://michaelshao.com)
 
 <section id="Getting-the-Job" />
 ## Getting the Job
 
+<section id="Applying-to-Jobs" />
+### Applying to Jobs
+
+If you're a graduating co-op student, attending Waterloo or elsewhere, you already know how this works, for the most part.
+Here are a few quick resource refreshers for finding awesome companies to work at.
+
+* [AngelList](https://angel.co) - AngelList has a wealth of early to late-stage (well-funded) startups that are always looking for talent. Post a talent profile and search for that full-time job.
+* [LinkedIn Premium](https://premium.linkedin.com) - LinkedIn itself already has some great tools; adding a premium search functionality to that experience can only enhance your search.
+
+Note that even as a co-op student, it can take a while (up to a few months) to find a full-time position if you end up not returning to a previous employer, for whatever reason.
+Keep applying, keep learning, keep trying - persistence and confidence is key. You will find a job in time - everyone does, in some way or another.
+(This is coming from someone who took three months to find a job, and ended up at Amazon.)
```

Comment:
Maybe not, but if this is only geared toward co-op, then I agree with your statement.

In the case of "regular" folk, or people who are graduating who don't have the same work experience as you or I, or the grades to get interviews, it might be harder. Job applications are a grind - this is advice I would give anyone who just graduated university and is in the position I was in during the months of January - March.